WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsActivity.kt

```diff
@@ -50,12 +50,10 @@ class StatsActivity : LocaleAwareActivity() {
         return super.onOptionsItemSelected(item)
     }
 
-    override fun onNewIntent(intent: Intent?) {
-        intent?.let {
-            val siteId = intent.getIntExtra(WordPress.LOCAL_SITE_ID, -1)
-            if (siteId > -1) {
-                viewModel.start(intent, restart = true)
-            }
+    override fun onNewIntent(intent: Intent) {
```

Comment:
I'm not sure why, but I'm getting a warning in Android Studio 🤔 :

```
Overriding method should call `super.onNewIntent`
```

However, the breakpoint was reached, and the `Intent` was non-null 🤷‍♂️ , and I don't think this is an issue with this PR.

Reply:
Yeaaa @mkevins , I think we already (briefly) discussed about that. From the looks of it, to my understanding, since this is also happening on `trunk`, this an IDE bug and I have already provided feedback on this warning, that this is a `False Positive`. I am not sure what else we can do here, or why this warning is shown on some of those `onNewIntent(...)` override methods and not on some others... 🤷

FYI: Personally, I chose to ignore this for now. ☺️

Reply:
> I think we already (briefly) discussed about that.

I recall discussing a similar issue, but I don't remember encountering this specific issue before. 🤷‍♂️  In any case, if it's on `trunk`, it's certainly not related with this PR (as I already suspected).